Assets/Scripts/SettingsMenu.cs

```diff
@@ -0,0 +1,30 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+using UnityEngine.Audio;
+using UnityEngine.UI;
+
+public class SettingsMenu : MonoBehaviour
+{
+
+    public AudioMixer audioMixer;
```

Comment:
does this need to be public? is this property accessed by any class outside of SettingsMenu? if not, we should make it a private with the SerializedField attribute ([reference documentation](https://docs.unity3d.com/ScriptReference/SerializeField.html))